service/controller/clusterapi/v20/resources/configmap/desired.go

```diff
@@ -209,7 +238,8 @@ func ingressControllerValues(configMapValues ConfigMapValues) ([]byte, error) {
 				UseProxyProtocol: configMapValues.IngressController.UseProxyProtocol,
 			},
 			Migration: IngressControllerGlobalMigration{
-				Enabled: configMapValues.IngressController.MigrationEnabled,
+				// Enabled is false because all providers have been migrated.
+				Enabled: false,
```

Comment:
Can this then be dropped or do we expect more migrations in the future?

Reply:
Yes this was the migration from k8scc to the helm chart. This is the chart default and there was more migration logic that could be removed.